lib/rules/no-cross-package-imports.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const CROSS_PACKAGE_IMPORT = /^ckeditor5/;
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'Disallow cross-package imports from CKEditor5 in specified packages.',
+			category: 'CKEditor5'
+		},
+		schema: []
+	},
+	create( context ) {
+		return {
+			ImportDeclaration: node => {
+				const targetPackages = context.settings.disallowedCrossImportsPackages;
```

Comment:
* This one can be moved before the `return {};` block (L20).
* Also, it would be good to check whether this value is specified. If not, use an empty array as default.